test/e2e-tests.sh

```diff
@@ -85,6 +85,18 @@ toggle_feature autocreate-cluster-domain-claims true config-network || fail_test
 go_test_e2e -timeout=2m ./test/e2e/domainmapping ${E2E_TEST_FLAGS} || failed=1
 toggle_feature autocreate-cluster-domain-claims false config-network || fail_test
 
+toggle_feature dataplane-trust enabled config-network || fail_test
+toggle_feature "logging.enable-request-log" true config-observability || fail_test
+toggle_feature "logging.request-log-template" 'TLS: {{.Request.TLS}}' config-observability || fail_test
+# with current implementation, Activator must be restarted when configuring system-internal-tls. See https://github.com/knative/serving/issues/13754
+restart_pod ${SYSTEM_NAMESPACE} "app=activator"
+go_test_e2e -timeout=2m ./test/e2e/internalencryption ${TEST_OPTIONS} || failed=1
```

Comment:
```suggestion
go_test_e2e -timeout=2m ./test/e2e/internalencryption ${E2E_TEST_FLAGS} || failed=1
```

Ugh... Sorry I missed this in the previous review. I think the [e2e skips](https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_serving/14092/kourier-stable_serving_main/1706461520491188224) the test due to this.